### PR TITLE
changed release name from 1.22 to 1.21

### PR DESCRIPTION
**Pull request by Shuayb1**
## Reviewers

- [ ] Ambient
- [ X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

**Comment by kfaseela** [comment]
could you please fix the CLA ?

File: content/en/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ average and tail latencies. The actual tail latency depends on the traffic patte
 ### Latency for Istio 1.21
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
+proxy. In the default configuration of Istio 1.21 (i.e. Istio with telemetry v2),
```
Comment (kfaseela) [inline content/en/docs/ops/deployment/performance-and-scalability/index.md:107]: why do we need 1.21 here? the page is not yet updated for 1.22 ?

Comment (Shuayb1) [reply content/en/docs/ops/deployment/performance-and-scalability/index.md:107]: yes the page is not yet updated to 1.22, it should be updated after 1.22.1 release

**Review by kfaseela — APPROVED** [review]

**Comment by ericvn** [comment]
I'm fine forcing the page to have the version. This will prevent the prior change where the reader would read the page an expect that the data is for the current version.

**Pull request merged**